level.go

```diff
@@ -105,3 +121,41 @@ func (l *Level) UnmarshalText(text []byte) error {
 	}
 	return nil
 }
+
+// Enabled return true if the given level is at or above this level.
+func (l Level) Enabled(lvl Level) bool {
+	return lvl >= l
+}
+
+// DynamicLevel creates an atomically changeable dynamic logging level.  The
+// returned level can be passed as a logger option just like a concrete level.
+//
+// The value's SetLevel() method may be called later to change the enabled
+// logging level of all loggers that were passed the value (either explicitly,
+// or by creating sub-loggers with Logger.With).
```

Comment:
nice - this is a much clearer explanation than the parent/child terminology I was using.